bokeh/core/compat/bokeh_renderer.py

```diff
@@ -147,7 +147,10 @@ def open_axes(self, ax, props):
 
     def close_axes(self, ax):
         "Complete the axes adding axes-dependent plot props"
-        background_fill_color = convert_color(ax.get_axis_bgcolor())
+        if LooseVersion(mpl.__version__) <= '1.5.9':
```

Comment:
`LooseVersion()` aside, is this really a breaking change in a version differing in patch/build number? Shouldn't this be `< 2`?